SevenZip/ArchiveUpdateCallback.cs

```diff
@@ -727,9 +727,6 @@ public void SetOperationResult(OperationResult operationResult)
                         _wrappersToDispose.Add(_fileStream);
                     }                                
                 _fileStream = null;
-                GC.Collect();
-                // Issue #6987
```

Comment:
For posterity, issue 6987 is from the original CodePlex project, titled `ContextSwitchDeadlock `.